File: C#/Lv_0/중복된 문자 제거.cs
using System;
using System.Linq;

public class 중복된_문자_제거 {
    public string solution(string my_string) {
        string answer = "";
        char[] s = my_string.Distinct().ToArray();
        for(int i=0; i<s.Length; i++) answer+= s[i];

        //string answer = string.Concat(my_string.Distinct()); // 2번 방법
        return answer;
    }
}

File: JavaScript/Lv_0/중복된 문자 제거.js
function solution(my_string) {
    var answer = '';
    for(let i of my_string) if(answer.indexOf(i) == (-1)) answer += i;
    return answer;
}

// 더 간단한 방법, Set을 이용한다.
// function solution(my_string) {
//     return [...new Set(my_string)].join('');
// }